File: Programacion Modulo Compras/SAE-COMPRAS/CapaDatos/DConexion.vb
Imports System.Collections.Generic
Imports System.Data.SqlClient
Imports System.Linq
Imports System.Text
Imports System.Threading.Tasks

Public Class DConexion
	'public static String cn = "Data Source=DESKTOP-U4KBH5U\SQLEXPRESS;Initial Catalog=BDEmpresa;Integrated Security=True";
	Public Shared CnBDEmpresa As [String] = "Data Source=CARLOS_ESPINOSA\SQLEXPRESS;Initial Catalog=BDEmpresa;Integrated Security=True"
	Public Shared CnMaster As [String] = "Data Source=CARLOS_ESPINOSA\SQLEXPRESS;Initial Catalog=BDEmpresa;Integrated Security=True"

	Public Function ChequearConexion() As [String]
		Dim mensaje As [String] = ""
		Dim SqlConexion As New SqlConnection()

		Try
			SqlConexion.ConnectionString = DConexion.CnBDEmpresa
			SqlConexion.Open()
			mensaje = "Y"
		Catch ex As Exception
			mensaje = ex.Message
		Finally
			SqlConexion.Close()
		End Try

		Return mensaje
	End Function
End Class

File: Programacion Modulo Compras/SAE-COMPRAS/CapaDatos/DConexion.cs
using System;
using System.Collections.Generic;
using System.Data.SqlClient;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace CapaDatos
{
    public class DConexion
    {
  
        public static String CnBDEmpresa = "Data Source=168.234.74.21;Initial Catalog=DB3;User ID=db3admin;Password=db3admin$";
        public static String CnMaster = "Data Source=168.234.74.21;Initial Catalog=DB3;User ID=db3admin;Password=db3admin$";

        public String ChequearConexion()
        {
            String mensaje = "";
            SqlConnection SqlConexion = new SqlConnection();

            try
            {
                SqlConexion.ConnectionString = DConexion.CnBDEmpresa;
                SqlConexion.Open();
                mensaje = "Y";
            }
            catch (Exception ex)
            {
                mensaje = ex.Message;
            }
            finally
            {
                SqlConexion.Close();
            }

            return mensaje;
        }
    }

}